templates/telegraf.conf.erb

```diff
@@ -21,6 +21,15 @@
   flush_jitter = "<%= @flush_jitter %>"
   precision = "<%= @precision %>"
   logfile = "<%= @logfile %>"
+<% if @logfile_rotation_interval -%>
```

Comment:
can you please implement thats for the options that verify that the content is placed into the template?

Reply:
Yes, I set these to use the default values in the params file instead of checking if they were passed.